### PR TITLE
features/extended-handler-responses: Added feature of extended respon…

### PR DESCRIPTION
**Pull request by daniil-grois**
…ses of handlers.

### CODE REVIEW

File: rapidy/_annotation_extractor.py [hunk 1]
```diff
@@ -240,6 +242,14 @@ class EndpointHandlerInfo(NamedTuple):
             """,
         ),
     ]
+    response_attr_name: Annotated[
+        Optional[str],
+        Doc(
+            """
+            Name of the attribute expecting the current `web.Response`.
```
Comment (LevZaplatin) [inline rapidy/_annotation_extractor.py:246]:
```suggestion
            Name of the attribute expected from the current `web.Response`.
```

File: rapidy/_annotation_extractor.py [hunk 2]
```diff
@@ -248,6 +258,14 @@ class EndpointHandlerInfo(NamedTuple):
             """,
         ),
     ]
+    return_annotation: Annotated[
+        Any,
+        Doc(
+            """
+            Handler return annotation.
```
Comment (LevZaplatin) [inline rapidy/_annotation_extractor.py:250]:
```suggestion
            Handler of a return annotation.
```
?

File: rapidy/_request_handlers.py [hunk 3]
```diff
@@ -126,20 +153,162 @@ async def _get_validated_data(
...
+    __slots__ = 'request_attribute_name', 'response_attribute_name', 'request_parameter_handlers', 'response_handler'
+
     def __init__(
             self,
             request_attribute_name: Annotated[
                 Optional[str],
                 Doc(
                     """
-                    The name of the handler attribute that expects the incoming `web.Request`.
+                    The name of the handler attribute that expecting the incoming `web.Request`.
```
Comment (LevZaplatin) [inline rapidy/_request_handlers.py]:
```suggestion
                    The name of the handler attribute that expects an incoming `web.Request`.
```

File: rapidy/_request_handlers.py [hunk 4]
```diff
@@ -126,20 +153,162 @@ async def _get_validated_data(
...
+                    """,
+                ),
+            ],
+            response_attribute_name: Annotated[
+                Optional[str],
+                Doc(
+                    """
+                    The name of the handler attribute that expecting the current `web.Response`.
```
Comment (LevZaplatin) [inline rapidy/_request_handlers.py]:
```suggestion
                    The name of the handler attribute that expects the current `web.Response`.
```

File: rapidy/_request_handlers.py [hunk 5]
```diff
@@ -159,9 +328,19 @@ def __init__(
                     """,
                 ),
             ],
+            response_handler: Annotated[
+                HTTPResponseHandler,
+                Doc(
+                    """
+                    The http response handler.
```
Comment (LevZaplatin) [inline rapidy/_request_handlers.py]:
```suggestion
                    Handler of a HTTP response.
```

File: rapidy/_request_handlers.py (hunk 3 above)
Comment (daniil-grois) [reply rapidy/_request_handlers.py]: done

File: rapidy/_request_handlers.py (hunk 5 above)
Comment (daniil-grois) [reply rapidy/_request_handlers.py]: done

File: rapidy/_annotation_extractor.py (hunk 2 above)
Comment (daniil-grois) [reply rapidy/_annotation_extractor.py:250]: no

**Pull request merged**